commerce_coordinator/apps/commercetools/clients.py

```diff
@@ -531,7 +531,7 @@ def create_return_payment_transaction(
         try:
             logger.info(
                 f"[CommercetoolsAPIClient] - Creating refund transaction for payment with ID {payment_id} "
-                f"following successful refund {refund['id']} in PSP: {psp}"
+                f"following successful refund {refund.get('id')} in PSP: {psp}"
```

Comment:
Did you confirm if it is a dict object?

Reply:
yes it is

Reply:
yes it should be as per our refund object. Its just a safeguard anyway